### DISCUSSION TITLE
JRE only download option missing

### DISCUSSION THREAD

**Discussion by erezbatish-ms** [microsoft/openjdk#603] [Q&A] [answered]
Hi, 
A customer is asking:
Red Hat, Ubuntu, AWS Corretto, and OpenJDK themselves provide just a JRE download option instead of the full JDK.
This JRE is all one is need to run Java apps, instead of the full JDK.
We do not see Microsoft offers an OpenJDK-JRE-<Version> only download option.

Any way this can be fixed?

Thanks

**Comment by brunoborges** [comment, marked as answer]
Please refer to this documentation article: https://learn.microsoft.com/en-us/java/openjdk/java-jlink-runtimes

**Reply by erezbatish-ms** [reply]
Customer is asking if we use jlink to create a JRE out of MS OpenJDK build – will MS support the JRE we created?

**Reply by karianna** [reply]
If it runs on Azure, they have commercial support from us for free, else they should contact openjdk-support@microsoft.com to discuss further.

**Reply by brunoborges** [reply]
> Customer is asking if we use jlink to create a JRE out of MS OpenJDK build – will MS support the JRE we created?

@erezbatish-ms in short, the answer is Yes.
